db/tables/operations/select.py

```diff
@@ -117,6 +117,10 @@ def get_table_description(oid, engine):
 def get_joinable_tables(
         engine, metadata, base_table_oid=None, max_depth=3, limit=None, offset=None
 ):
+    """
+    Output presumed to be a set of tuples that look like this: `(base table oid, target table oid,
+    join parameter path, foreign key path, depth, multiple results boolean flag).
+    """
```

Comment:
Not sure why you say that it is incorrect. Are you referring to me saying that the data structure is a set of tuples? I meant that in the most general sense.

Reply:
Note to self: be more specific. It's a collection of ordered-dicts/sa-rows with specific column names.

Reply:
@mathemancer I agree about the perspective of knowing, in that it would be nice to know with certainty and to document that. I wrote it from a place of not knowing with certainty. I think that's fine, as long as you're upfront about not knowing. As my knowledge grows, I would update the docstring, and I expect others to do the same.